pkg/security/scc/byrestrictions.go

```diff
@@ -35,9 +36,19 @@ const (
 	runAsRangePoints   points = 20000
 	runAsUserPoints    points = 10000
 
+	runCapsDefaultPoints  points = 5000
+	runCapsAddOnePoints   points = 300
```

Comment:
Why do we have a different scores for the cases when cap is added and when it's removed? Do we have a case when we want to treat them differently?